api/tests/core/bookings/test_api.py

```diff
@@ -528,7 +528,7 @@ def test_raise_if_already_cancelled(self):
         assert booking.status is BookingStatus.CANCELLED
         assert booking.cancellationReason == BookingCancellationReasons.BENEFICIARY  # unchanged
 
-        assert push_testing.requests == []
+        assert not push_testing.requests
```

Comment:
Ici ce sera vrai si `push_testing.requests==[]` et si `push_testing.requests is None`

Peut être que ce n'est pas grave, mais dans l'idée de garder le test iso : 
```diff
- assert not push_testing.requests
+ assert len(push_testing.requests) == 0
```
doit éviter le warning non ?
Low logaf

Reply:
"On sait" que `push_testing.requests` est une liste, ça ne peut pas être `None`. C'est défini comme une liste et on ne fait que des append dessus. Ok, tout est possible, mais je pense que si jamais ça devient `None`, y a pas mal d'autres tests qui vont casser ! ;)

Reply:
Ok pour moi :)